.github/ISSUE_TEMPLATE/02-FEATURE-REQUEST.yml

```diff
@@ -0,0 +1,52 @@
+name: "\U0001F4A1 Feature Request"
+description: "Suggest a new feature or enhancement"
+labels: ["Type: Feature Request"]
```

Comment:
What will happen if this label doesn't exist in the repo? It is one of the labels that we use in most of the repos, but not the default one, that is present in all new GitHub repos

Reply:
@tom-borcin If the label is not in the repository, then no label is assigned (no crash or error occurs).

By the way, we can (and probably should) set the labels at the organizational level under `Settings -> Repository -> General -> Repository labels`. This will ensure consistent labels across all Espressif projects, rather than relying on the "Create project tools" helper (we can keep it as fallback).

Reply:
Given that we will be using `Type: Feature Request` and `Type: Bug` labels in these default templates, it makes sense to set them on organizational level.

Reply:
@tom-borcin I added the Espressif default labels at the organization level, so they will automatically be available in every new project, whether created by the Create Project tools or manually in GitHub. 

I didn't add the <kbd>**Status: ...**</kbd> and <kbd>**Resolution: ...**</kbd> labels, as these are created by Jira automation when needed.

Default GitHub labels that we are replacing with our own (<kbd>bug</kbd>, <kbd>documentation</kbd>, <kbd>duplicate</kbd>, <kbd>enhancement</kbd>, <kbd>invalid</kbd>, <kbd>wontfix</kbd>) were removed - this will not have any effect on existing repositories.

Eventually, I will remove the feature from the **Create-Project-tools** that automatically creates these labels, as it is no longer necessary for new projects (will keep it only in the "Manage GitHub" workflow). Until then, if a label with the same name is already present, the **Create-Project-tools** will recognize and ignore it while creating a new project. 

There is no conflict, and the process should be safe.

<img src="https://github.com/user-attachments/assets/b8c05d9a-2bf6-4a95-8501-9036e5ca02be" width=500px>